internal/cmd/stack/stack_selector.go

```diff
@@ -47,6 +62,29 @@ func getStackID(cliCtx *cli.Context) (string, error) {
 	return got, nil
 }
 
+func stackExists(ctx context.Context, stackId string) error {
+	var query struct {
+		Stack struct {
+			ID string `graphql:"id"`
+		} `graphql:"stack(id: $id)"`
+	}
+
+	variables := map[string]interface{}{
+		"id": graphql.ID(stackId),
+	}
+
+	err := authenticated.Client.Query(ctx, &query, variables)
+	if err != nil {
+		return fmt.Errorf("failed to query GraphQL API: %w", err)
```

Comment:
Let's be more descriptive here:
```suggestion
		return fmt.Errorf("failed to query GraphQL API when checking if a stack exists: %w", err)
```